bindings/python/benchmarks/ctc.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import torch
+import torch.nn
+import time
+import gtn
+import gtn.criterion
+
+
+parser = argparse.ArgumentParser("CTC Benchmark with GTNs")
```

Comment:
There is a `time_utils.py` which you can use to do some of the basics in timing (warmup and printing results etc). You can also add the cuda synchronize there since it should be part of any timing.